.applier/group_vars/seed-hosts.yml

```diff
@@ -55,6 +56,12 @@ test_pipelines:
       PIPELINE_FILENAME: Jenkinsfile.test
       PIPELINE_SOURCE_REPOSITORY_REF: "{{ slave_repo_ref }}"
       PIPELINE_SOURCE_REPOSITORY_URL: "{{ repository_url }}"
+    graal:
+      NAME: jenkins-slave-graal
```

Comment:
Let's be consistent and use `graalvm` across the board

Reply:
Done